src/platform/OpenThread/GenericNetworkCommissioningThreadDriver.h

```diff
@@ -106,6 +106,7 @@ class GenericThreadDriver final : public ThreadDriver
 
     CHIP_ERROR CommitConfiguration() override;
     CHIP_ERROR RevertConfiguration() override;
+    CHIP_ERROR DisconnectDriver() override;
```

Comment:
`DisconnectNetwork()` would go well with `ConnectNetwork()` below.

Reply:
Also, I think it would make sense to define this in `WirelessDriver`? Or how/where do you propose to implement the logic that can check if the currently active connection is no longer in the `Networks` list?

Reply:
Added this change to https://github.com/project-chip/connectedhomeip/pull/35256, will close this later.